src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -193,4 +193,13 @@ Temporarily using the Windows Terminal default settings.
   <data name="SettingsMenuItem" xml:space="preserve">
     <value>Settings</value>
   </data>
-</root>
+  <data name="Cancel" xml:space="preserve">
+    <value>Cancel</value>
+  </data>
+  <data name="Close all" xml:space="preserve">

```

Comment:
I don't think we should have a resource name with a space in it. This should probably be CloseAll and update the code below to call it the same way.

Reply:
changed